tools/sgen/input/FluentUIStyle.yml

```diff
@@ -440,12 +436,27 @@ AP_MSFListCellTokens:
   labelColor: $Colors.Foreground.neutral1
   labelFont: $Typography.body
   leadingViewColor: $Colors.Foreground.neutral4
-  leadingViewSize: [ small: $Icon.size.xSmall, medium: $Icon.size.medium, large: $Icon.size.xxlarge]
+  leadingViewSize: [ small: $Icon.size.xSmall, medium: $Icon.size.medium, large: $Icon.size.xxlarge ]
   sublabelColor: $Colors.Foreground.neutral3
   sublabelFont: $Typography.caption1
   trailingItemForegroundColor: $Colors.Foreground.neutral3
   trailingItemSize: $Icon.size.medium
 
+AP_MSFHeaderFooterTokens:
```

Comment:
FYI: With the new changes I have merged today, there should be a new generated file for MSFHeaderFooterTokens that will contain these tokens and the MSFPrimaryHeaderTokens too.

That's one of the sources of the conflicts of this branch.